pkg/cmd/operator/sidecar.go

```diff
@@ -197,10 +198,10 @@ func (o *SidecarOptions) Run(streams genericclioptions.IOStreams, cmd *cobra.Com
 	// Wait for the service that holds identity for this scylla node.
 	serviceFieldSelector := fields.OneTermEqualSelector("metadata.name", o.ServiceName)
 	serviceLW := &cache.ListWatch{
-		ListFunc: func(options metav1.ListOptions) (runtime.Object, error) {
+		ListFunc: helpers.UncachedListFunc(func(options metav1.ListOptions) (runtime.Object, error) {
```

Comment:
Is there any reason why skipping cache is required in sidecar? We are not patching here anything and right after waiting until change is rolled out, so i guess cache shouldn't be an issue.

Reply:
in theory it could see any arbitrarily old object. although unlikely it could be a previous scyllacluster with different UUID